<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewSettingsWidget</class>
 <widget class="QWidget" name="ViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="config_tab">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <widget class="QCheckBox" name="showTagHistory">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Tag History</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showGeoFencingMode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>109</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Geo-Fencing Mode</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="zone1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="zone2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_z1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Zone 1 (m)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_z2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Zone 2 (m)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="inAlarm">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Alarm Inside</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="outAlarm">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Alarm Outside</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="tagHistoryN">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="showTagTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Tag Table</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showAnchorTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Anchor Table</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showAnchorTagCorrectionTable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>251</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Anchor-Tag Correction Table</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showNavigationMode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tracking/Navigation Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="filtering">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>260</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_z1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtering options:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="useAutoPos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Auto - Positioning</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_logingstatus">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Logging:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logging_pb">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>300</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_logingstatus_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_logfile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Log filename</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_logingstatus_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="floorplan_tab">
      <attribute name="title">
       <string>Floor Plan</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>pixels</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="floorplanXScale_sb">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>pixesl/m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X Offset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="floorplanXOff_sb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>-65536</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="floorplanPath_lb">
         <property name="text">
          <string>No Floor Plan</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="floorplanFlipY_cb">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>pixels</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="floorplanYScale_sb">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>pixels/m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="floorplanYOff_sb">
         <property name="minimum">
          <number>-65536</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>X Scale</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Flip X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y Offset</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="floorplanFlipX_cb">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y Scale</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Flip Y</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="showOrigin">
           <property name="text">
            <string>Show Origin</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="scaleY_pb">
           <property name="text">
            <string>Y Scale</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="scaleX_pb">
           <property name="text">
            <string>X Scale</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="origin_pb">
           <property name="text">
            <string>Set Origin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="floorplanOpen_pb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="saveFP">
         <property name="text">
          <string>Save Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="grid_tab">
      <attribute name="title">
       <string>Grid</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>21</y>
         <width>143</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Width (m)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="gridWidth_sb">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Height (m)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="gridHeight_sb">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="gridShow">
          <property name="text">
           <string>Show Grid</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gridWidth_sb</tabstop>
  <tabstop>gridHeight_sb</tabstop>
  <tabstop>floorplanOpen_pb</tabstop>
  <tabstop>floorplanXOff_sb</tabstop>
  <tabstop>floorplanYOff_sb</tabstop>
  <tabstop>floorplanXScale_sb</tabstop>
  <tabstop>floorplanYScale_sb</tabstop>
  <tabstop>floorplanFlipX_cb</tabstop>
  <tabstop>floorplanFlipY_cb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
